client/src/assets/icons/green-not-completed.tsx

```diff
@@ -5,11 +5,15 @@ function GreenNotCompleted(
   props: JSX.IntrinsicAttributes & React.SVGProps<SVGSVGElement>
 ): JSX.Element {
   const { t } = useTranslation();
-
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const suppressSrOnly = props['data-suppress-sronly'];
   return (
     <>
-      <span className='sr-only'>{t('icons.not-passed')}</span>
+      {suppressSrOnly !== 'true' && (
+        <span className='sr-only'>{t('icons.not-passed')}</span>
+      )}
```

Comment:
```suggestion
      {suppressSrOnly && (
        <span className='sr-only'>{t('icons.not-passed')}</span>
      )}
```
It'll be boolean already.